docs/source/flow_setup.rst

```diff
@@ -235,3 +235,39 @@ You may define user-specific config parameters as follows
 ::
 
     cp flow/core/config.template.py flow/core/config.py  # Create template for users using pycharm
+
+
+
+====================
+VNC desktop using Docker for simple utilisation
```

Comment:
I don't think VNC is a standard word, can you add something like "Remote desktop
?

Reply:
@eugenevinitsky  Changed